PowerEditor/installer/nativeLang/croatian.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
 	Croatian localization for Notepad++
-	Updated: April 24, 2025 by Elvis Gambiraža (el.gambo@gmail.com)
+	Updated: May 30, 2025 by Elvis Gambiraža (el.gambo@gmail.com)
 	All the comments are for explanation, they are not for translation.
 -->
 <NotepadPlus>
-	<Native-Langue name="Hrvatski" filename="croatian.xml" version="8.8.0">
+	<Native-Langue name="Hrvatski" filename="croatian.xml" version="8.8.2">
```

Comment:
Please don't guess the version number without the official announcement.
Use the current version number instead.